lib/foreman_maintain/concerns/disk_utility.rb

```diff
@@ -0,0 +1,41 @@
+require 'foreman_maintain/concerns/disk_utility/device'
+require 'foreman_maintain/concerns/disk_utility/disk_type'
+require 'foreman_maintain/concerns/disk_utility/disk_io'
+require 'foreman_maintain/concerns/disk_utility/file_io'
+
+module ForemanMaintain
+  module Concerns
+    module DiskUtility
+      include Logger
+      MIN_SPEED_MBPS = 80
+      DEFAULT_DIRS = ['/var/lib/pulp', '/var/lib/mongodb', '/var/lib/pgsql'].freeze
+
+      def self.included(klass)
+        klass.extend(self)
+      end
+
+      def pass?
```

Comment:
A concern should not define this kind of generic method, that could be easily mistaken/conflicted with some other concern. Perhaps, instead of having this as a concern, the logic that this module defines could go directly to the check.

The the `Concerns::DiskUtility` module, that groups the relevnat classes could be renamed to `Utils::Disk` (while introducing the `Utils` module: I see a benefit of having such a module here and we will utilize that later for sure for other things as well